src/Components/WebView/WebView/src/IpcSender.cs

```diff
@@ -29,7 +29,12 @@ public void ApplyRenderBatch(long batchId, RenderBatch renderBatch)
         {
             renderBatchWriter.Write(in renderBatch);
         }
-        var message = IpcCommon.Serialize(IpcCommon.OutgoingMessageType.RenderBatch, batchId, Convert.ToBase64String(arrayBuilder.Buffer, 0, arrayBuilder.Count));
+
+        var data = arrayBuilder.Buffer.AsSpan(0, arrayBuilder.Count).ToArray();
+        var message = IpcCommon.Serialize(IpcCommon.OutgoingMessageType.RenderBatch,
+            batchId,
+            data);
```

Comment:
I do not think that this by itself will be enough, the JS code likely has to react to this.

Reply:
It is binary the same, for the consumer side (JS side) it looks the same. So there is no JS adaption needed?
If the Base64 encoding is done before calling Serialize, or the Serializer itself is doing that for us makes no difference.